actions/docker/cmd.go

```diff
@@ -0,0 +1,42 @@
+package docker
+
+import (
+	"github.com/codegangsta/cli"
+	"github.com/deis/deisrel/actions"
+	dlib "github.com/fsouza/go-dockerclient"
+	"github.com/google/go-github/github"
+)
+
+// Command returns the entire set of subcommands for the 'deisrel docker ...' command
+func Command(ghClient *github.Client, dockerCl *dlib.Client) cli.Command {
+	return cli.Command{
+		Name: "docker",
+		Subcommands: []cli.Command{
+			cli.Command{
+				Name:        "retag",
+				Description: "This command pulls specific Docker images for each corresponding repository's Git SHA, then retags each one to a uniform release tag",
+				Usage:       "Retag each specific Docker image from a repo-specific Git SHA to a uniform release tag",
+				Flags: []cli.Flag{
```

Comment:
@arschles can we also allow use of the `--ref <ref>` [flag](https://github.com/deis/deisrel/blob/09a1b5b0ae047d19cba62c74aee181d26005ad26/actions/common.go#L23-L24) for this command so that the shas can be pulled from HEAD of a branch other than `master`?